src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('TBODY');
+const rows = document.querySelectorAll('TR');
+
+const appendColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+const appendrowButton = document.querySelector('.append-row');
+const removerowButton = document.querySelector('.remove-row');
+
+appendColumnButton.addEventListener('click', () => {
+  [...table.children].forEach(
+    el => el.append(el.lastElementChild.cloneNode(true))
+  );
+
+  if (rows[0].children.length >= 10) {
+    appendColumnButton.disabled = true;
+  }
+
+  if (rows[0].children.length >= 3) {
+    removeColumnButton.disabled = false;
+  }
+});
+
+removeColumnButton.addEventListener('click', () => {
+  [...table.children].forEach(
+    el => el.lastElementChild.remove()
+  );
+
+  if (rows[0].children.length <= 2) {
+    removeColumnButton.disabled = true;
+  }
+
+  if (rows[0].children.length < 10) {
+    appendColumnButton.disabled = false;
+  }
```

Comment:
You repeat this code. Better to write a function that you can use for both rows and columns.